src/site/stages/build/process-cms-exports/tests/transformed-entities/paragraph-list_of_link_teasers.json

```diff
@@ -0,0 +1,89 @@
+{
+  "contentModelType": "paragraph-list_of_link_teasers",
+  "entity": {
+    "entityBundle": "list_of_link_teasers",
+    "entityType": "paragraph",
+    "parentFieldName": "",
+    "fieldTitle": "Tools to help you manage your health if you're registered as a patient in a VA health facility",
+    "fieldVaParagraphs": [
+      {
+        "contentModelType": "paragraph-link_teaser",
+        "entity": {
+          "fieldLink": {
+            "url": {
+              "path": "entity:node/27"
```

Comment:
Yeahhhhh...I'm guessing we'll need to do something about this at some point. You wanna write up a ticket for this, or shall I?

Reply:
I'll do it.